### DISCUSSION TITLE
How i can invert the image?

### DISCUSSION THREAD

**Discussion by tsugl** [peterhinch/micropython-nano-gui#51] [Q&A]
Hello, i use display LS027B4DH01. How i can invert the image? If this function is not implemented, how i can it be properly integrated into the code?
![photo_2023-05-03_09-04-45](https://user-images.githubusercontent.com/55076360/235831372-c71a124e-5cf5-490d-8dc7-290697ad52aa.jpg)

**Comment by peterhinch** [comment]
This requires a modification to the driver. Normally it can be done by reversing the order of the line output, and also reversing the bit order of the bits in each row. The Sharp displays seem to be little used, so I have no plans to enhance the driver.
